random.go

```diff
@@ -0,0 +1,63 @@
+//
```

Comment:
@jodh-intel  I wonder if this file should be called `random_linux.go`
see https://github.com/kata-containers/runtime/issues/379

Reply:
Good call!

/cc @raravena80

Reply:
kata-containers/runtime#379 is about building `kata-runtime` on Mac. Are we planning on building `kata-agent` on other platforms as well? IMO kata-agent relies heavily on Linux specific features (sysfs, device tree, cgroups, namespaces etc.), and we have not done any thing/design to make it cross platform AFAICT.

Reply:
@bergwolf - I agree that it could be an uphill struggle to port the agent. But I still think it makes sense to put obviously Linux-specific code in a `_linux.go` file. It's not a lot of effort and might make life easier for @raravena80 :)

Reply:
@jodh-intel My point is that kata-agent on other platforms may never happen, unless we are willing to rewrite kata-agent completely (in which case hyperstart might be better suited for the task), -- e.g., libcontainer does not work on platforms other than linux. I can rename this to random_linux.go but I wanted to remind @raravena80 (or anyone who wants to look at the direction) that it is a dead end.